src/main.cpp

```diff
@@ -59,6 +59,9 @@ DEFINE_bool(term_colors, false, "Use terminal colors syntax highlighting.");
 DEFINE_string(output_format, "tabular",
               "Query output format. Can be csv/tabular. If output format is "
               "other than tabular `fit-to-screen` flag is ignored.");
+DEFINE_bool(verbose_query, false,
```

Comment:
This was just an example at hand in the #35, but maybe a better flag name would be `--print-execution-time-info` or something similar which is aligned with the actual change, anyway, something to think about and maybe adjust 😄